gateway/rdb/src/main/kotlin/org/doorip/gateway/rdb/trip/TodoJpaEntity.kt

```
@@ -0,0 +1,45 @@
+@Table(name = "todo")
+@Entity
+internal class TodoJpaEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "todo_id", columnDefinition = "bigint", nullable = false)
+    var id: Long = 0
+        protected set
+
+    @Column(name = "title", columnDefinition = "varchar(255)", nullable = false)
+    lateinit var title: String
+        protected set
```

detekt (1.23.7)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)